File: LiveDataDemo/src/main/java/com/dale/livedatademo/model/TestViewModel.java
package com.dale.livedatademo.model;

import androidx.lifecycle.ViewModel;

import com.dale.livedatademo.ui.App;

public class TestViewModel extends ViewModel {

    public void getTestNetLiveData(){
        App.netLiveData.postValue("我艹你妹");
    }

}

File: app/src/main/java/com/dale/kotlin/viewmodel/TestViewModel.kt
package com.dale.kotlin.viewmodel;

import androidx.lifecycle.ViewModel
import com.ty.bwagent.api.Api
import com.ty.bwagent.bean.BaseEntity
import com.ty.net.NetSdk
import com.ty.net.bean.NetLiveData

class TestViewModel : ViewModel() {
    var mLiveData = NetLiveData<BaseEntity<String>>()

    fun test() {
        NetSdk.create(Api::class.java)
            .getPreInfo()
            .asJSONType()
            .send(mLiveData)
    }

}